src/pages/City/components/NearestCitiesWeatherWidget/NearestCitiesWeatherWidget.tsx

```diff
@@ -0,0 +1,28 @@
+import { CityWeatherShortcut } from 'components/CityWeatherShortcut';
+import { CityWeatherShortcutLoader } from 'components/CityWeatherShortcut/components/CityWeatherShortcutLoader';
```

Comment:
**suggestion** if this component is within public API it should be exported by CityWeatherShortcut index.ts